### PR TITLE
view_product_homepage

### PR DESCRIPTION
**Pull request by nminhduc**

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class StaticPagesController < ApplicationController
   def home
+    @products = Product.paginate page: params[:page], per_page: 12
```
Comment (thanhtt) [inline app/controllers/static_pages_controller.rb]: cho vào Settings

File: app/views/products/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <div class="row">
+      <div class="col-md-3">
+        <%= image_tag @product.image.thumb.url, class: "img-rounded" %>
+      </div>
+      <div class="col-md-3">
+      <span class="badge product_show"> <%= t "products.price" %>:
+        <%= @product.price %>
+      </span><br>
+      <span class="label label-success product_show">
+      <%= t "products.name" %>:<%= @product.name %>
```
Comment (thanhtt) [inline app/views/products/show.html.erb]: indent

File: config/initializers/config.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Config.setup do |config|
+  # Name of the constat exposing loaded settings
+  config.const_name = 'Settings'
```
Comment (dieunb) [inline config/initializers/config.rb]: Double quote

**Comment by thanhtt** [comment]
ok :v:

**Pull request merged**